### ISSUE TITLE
ChromeIPass with accounts.google.com

### ISSUE THREAD

**Issue by cawoodm** [pfn/passifox#462]
I read some people having unresolved issues with Google's 2-page authentication (https://accounts.google.com/ServiceLogin?service=mail) but even their workaround (Ctrl+Shift+U) is not working for me.

I have only one Google login and the URL is **https://accounts.google.com/**. The page loads with no fields filled out. Ctrl+Shift+U does nothing. The Right-Click option (sometimes) brings "#101 Cannot find fields to fill in". I then noticed the question mark on the extension's button and it asked me to choose a login - even though I only have one. Even once I choose a login, it still does not fill it out: neither immediately, nor with Ctrl+Shift+U nor with Right-Click. Yes, I did go through and identify the username and password fields.

So basically, as a new user, I'm finding it very difficult to see any benefit to the extension on this site and hard to see why it could not work.

I guess the issues are:
 - Why does it ask for a login for this page if I only have one?
 - Why does it not fill out logins automatically?
 - Why does Ctrl+Shift+U do nothing
 - Why does Right-Click -> Fill Username and Password do nothing?

Thanks

Marc

**Comment by pfn** [comment]
Fill username will not work if a password field is not found.

**Comment by bilogic** [comment]
Hi,

I have a few google accounts. Right now (2nd June 2016), Google uses ```#email``` and ```#passwd``` for their 2 page login.

I used ```Choose own credential fields for this page``` to detect ```#email``` and have to skip the password field because it is not visible yet. This works good.

I thought of some solutions:
1. Can we edit the ```Specified credential fields``` in the settings to allow adding ```#passwd``` manually?
So that on the 2nd page, when clicking on the chromeIPass icon, we can select the correct account 1 more time to fill in the password
1. On the 2nd page, I noticed that chromeIPass does not detect the password field (No key logo at the end of the field). Can we add option when right click on password field, chromeIPass->Set as password field, to tell chromeIPass that it is the password field and we can select the desired account from the chromeIPass icon?

**Comment by bilogic** [comment]
Hi,

I just realized, if you logout completely (https://www.google.com/accounts/Logout?service=wise) and at the email, press Ctrl-Shift-U, we get to choose an account from KeePass, after clicking on Next, the password is automatically entered.

Why does chromeIPass not detect the username field? Is it because of ```#email```?
